funcx_endpoint/funcx_endpoint/executors/high_throughput/funcx_worker.py

```diff
@@ -157,7 +157,8 @@ def start(self):
             if task_type == b'WRKR_DIE':
                 logger.info("*** WORKER {} ABOUT TO DIE ***".format(self.worker_id))
                 # Kill the worker after accepting death in message to manager.
-                exit()
+                sys.exit()
```

Comment:
Is it fine to do this? I think it should be `sys.exit()` here, correct?

Reply:
Yes. `exit()` is supposed to be used in interactive contexts.